runtime/common/src/apis/pools.rs

```diff
@@ -46,5 +46,7 @@ decl_runtime_apis! {
 		fn tranche_id(pool_id: PoolId, tranche_index: TrancheIndex) -> Option<TrancheId>;
 
 		fn tranche_currency(pool_id: PoolId, tranche_loc: TrancheLoc<TrancheId>) -> Option<Currency>;
+
+		fn nav(pool_id: PoolId) -> Option<(Balance, Nav<Balance>)>;
```

Comment:
@mustermeiszer Not sure if we would rather want to expose all data in a new type to be more explicit. Here we return `(nav.total(pool.reserve), nav)`.

Reply:
Would it be possible to use a structured type for this? Just because it's quite opaque now what the data returned here is (hard to know that the first element is the reserve and the second element is the NAV)

Reply:
Not at all. I wanted to propose the most simple version and then iterate on that based on the feedback.